spacy/util.py

```diff
@@ -187,6 +189,20 @@ def get_module_path(module: ModuleType) -> Path:
     return Path(sys.modules[module.__module__].__file__).parent
 
 
+def load_vectors_into_model(
```

Comment:
@adrianeboyd Do you know of anything else we need to do when loading the vectors, other than these things?

We no longer have to set the `lexeme.rank` stuff.